common/src/main/java/org/apache/celeborn/common/network/protocol/ChunkFetchFailure.java

```diff
@@ -49,7 +49,9 @@ public void encode(ByteBuf buf) {
   public static ChunkFetchFailure decode(ByteBuf buf) {
     StreamChunkSlice streamChunkSlice = StreamChunkSlice.decode(buf);
     String errorString = Encoders.Strings.decode(buf);
-    return new ChunkFetchFailure(streamChunkSlice, errorString);
+    ChunkFetchFailure chunkFetchFailure = new ChunkFetchFailure(streamChunkSlice, errorString);
+    chunkFetchFailure.setBody(buf);
```

Comment:
I think this change is not correct. Check the TransportFrameDecoder's logic below.
```
 public void channelRead(ChannelHandlerContext ctx, Object data) {
    ByteBuf in = (ByteBuf) data;
    buffers.add(in);
    totalSize += in.readableBytes();

    while (!buffers.isEmpty()) {
      ByteBuf frame = decodeNext();
      if (frame == null) {
        break;
      }
      Message msg = Message.decode(curType, frame);
      if (msg.body() == null) {
        frame.release();
      }
      ctx.fireChannelRead(msg);
      clear();
    }
  }
```

Reply:
Yea, will close this and next time meet this make a dump